resources/services/wafregional/rate_based_rules.go

```diff
@@ -0,0 +1,178 @@
+package wafregional
+
+import (
+	"context"
+
+	"github.com/aws/aws-sdk-go-v2/aws"
+	"github.com/aws/aws-sdk-go-v2/service/wafregional"
+	"github.com/aws/aws-sdk-go-v2/service/wafregional/types"
+	"github.com/cloudquery/cq-provider-aws/client"
+	"github.com/cloudquery/cq-provider-sdk/provider/diag"
+	"github.com/cloudquery/cq-provider-sdk/provider/schema"
+)
+
+//go:generate cq-gen -config=rate_based_rules.hcl -domain=wafregional -resource=rate_based_rules
+func RateBasedRules() *schema.Table {
+	return &schema.Table{
+		Name:                 "aws_wafregional_rate_based_rules",
+		Description:          "This is AWS WAF Classic documentation",
+		Resolver:             fetchWafregionalRateBasedRules,
+		Multiplex:            client.ServiceAccountRegionMultiplexer("waf-regional"),
+		IgnoreError:          client.IgnoreAccessDeniedServiceDisabled,
+		DeleteFilter:         client.DeleteAccountRegionFilter,
+		Options:              schema.TableCreationOptions{PrimaryKeys: []string{"account_id", "region", "id"}},
+		PostResourceResolver: postResolveTags,
```

Comment:
Why are we grabbing the tags as a `PostResourceResolver` rather than a column resolver?

Reply:
Because fetching tags requires ARN that we generate on the fly with the column resolver. Without duplicating ARN stuff post resource resolver is the only option, it seems.

I am open to any suggestions.

Reply:
Generating the ARN a second time doesn't sound too bad, so that the GetTags call can be parallelized.

Reply:
Alright, I'll fix it then.